pwndbg/lib/functions.py

```diff
@@ -57,9 +64,9 @@ def format_flags_argument(flags: tuple[Flag, ...], value: int):
     # descending popcount, this loop will output more specific
     # flags.
     for flag in flags:
-        if (value & flag.value) == flag.value:
-            flag_names.append(flag.name)
-            value = value & ~flag.value
+        if (value & int(flag)) == int(flag):
+            flag_names.append(str(flag))
+            value = value & ~int(flag)
```

Comment:
shall we maybe move int(flag) to resolve_constants? do we ever need pwnlib's Constant object?